opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -765,6 +765,18 @@ def save_as_pt(
 
         # save tokenizer.json in save_json_folder_name
         model.save(save_json_folder_path)
+        with open(save_json_folder_path + "/tokenizer.json") as user_file:
+            file_contents = user_file.read()
+        parsed_json = json.loads(file_contents)
```

Comment:
You can combine line 769-770 and write just `parsed_json = json.load(user_file)`. [`load` not `loads`]

Reply:
Done